packages/playground/src/explorer/views/nodes.vue

```diff
@@ -172,22 +172,22 @@ export default {
     const request = debounce(_requestNodes, 1000);
 
     watch(
-      mixedFilters,
+      MixedNodeFilters,
       async () => {
-        const queries = getQueries(mixedFilters.value);
+        const queries = getNodeQueries(MixedNodeFilters.value);
         await request(queries, { loadFarm: true });
       },
       { deep: true },
     );
 
     // The mixed filters should reset to the default value again..
-    const inputFiltersReset = (nFltrNptsVal: FilterInputs) => {
-      mixedFilters.value.inputs = nFltrNptsVal;
-      mixedFilters.value.options.status = NodeStatus.Up;
-      mixedFilters.value.options.gpu = undefined;
-      mixedFilters.value.options.gateway = undefined;
-      mixedFilters.value.options.page = 1;
-      mixedFilters.value.options.size = 10;
+    const inputFiltersReset = (nFltrNptsVal: FilterNodeInputs) => {
+      MixedNodeFilters.value.inputs = nFltrNptsVal;
+      MixedNodeFilters.value.options.status = NodeStatus.Up;
+      MixedNodeFilters.value.options.gpu = undefined;
+      MixedNodeFilters.value.options.gateway = undefined;
+      MixedNodeFilters.value.options.page = 1;
```

Comment:
page & size should be set in the table itself and could be passed here instead of being hardcoded.

Reply:
this is outside the scope of this PR as it is in explorer nodes and not farms , please create an issue for it